### PR TITLE
Add .travis.yml for Java tests

### PR DESCRIPTION
**Pull request by rdblue**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  - oraclejdk8
+
+install: mvn install --batch-mode -DskipTests=true -Dmaven.javadoc.skip=true -Dsource.skip=true > mvn_install.log || mvn install --batch-mode -DskipTests=true -Dmaven.javadoc.skip=true -Dsource.skip=true -V > mvn_install.log || (cat mvn_install.log && false)
+
+script: mvn test -Dmaven.main.skip=true
```
Comment (busbey) [inline .travis.yml]: might be worth a `--fail-never` followed by grepping the log for build failed?

**Comment by busbey** [comment]
+1 either as is, or with the change to build all of the modules each time.

please include a jira-id on commit.

**Comment by busbey** [comment]
Still doing this one?

**Comment by johnsgill3** [comment]
This looks like the only tests that will be run are the Java language ones. With support that @spacharya has done to get Apache Yetus integrated, is this still planned?

**Comment by Fokko** [comment]
Closing this one. Superseded by #353. If you're missing something @rdblue, feel free to open a new PR.

**Pull request closed without merging**